src/launch/linux/ioctl.rs

```diff
@@ -32,6 +33,8 @@ const ENC_OP: Ioctl<WriteRead, &c_ulong> = unsafe { KVM.write_read(0xBA) };
 
 /// Initialize the SEV platform context.
 pub const INIT: Ioctl<WriteRead, &Command<Init>> = unsafe { ENC_OP.lie() };
+/// Create encrypted guest context.
+pub const LAUNCH_START: Ioctl<WriteRead, &Command<LaunchStart>> = unsafe { ENC_OP.lie() };
```

Comment:
```suggestion

/// Create encrypted guest context.
pub const LAUNCH_START: Ioctl<WriteRead, &Command<LaunchStart>> = unsafe { ENC_OP.lie() };
```

My general rule of thumb is vertical space before comments and no space if there is no comments. I feel like this makes the comments easier to read.

Reply:
This is not a requirement but a preference.